packages/braid-design-system/src/lib/components/Autosuggest/reverseMatches.ts

```diff
@@ -0,0 +1,22 @@
+export type Match = [number, number];
+
+export function reverseMatches(suggestion: string, matches: Match[]): Match[] {
+  const suggestionLength = suggestion.length;
```

Comment:
Given the matches are zero based, feel like this should be `length - 1`?

E.g. in the tests above, reversing `[]` match on `Apple` produces `[0,5]`. Should that be `[0,4]`?